index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RESUME Roman Levsha</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="name">
+        <div class="container">
+          <div class="name__inf">
+          <h1>Levsha Roman</h1>
+          <p><em>JavaScript beginner programmer</em></</p>
+          </div>
+        </div>
+    </header>
+    <section class="basic">
+        <div class="container">
+          <div class="basic__inf">
+            <figure>
+              <img
+                src="/IMG_20200818_143736.jpg"
+                alt="Foto Roma"
+                width="190px">
+              <figcaption>R.Levsha.</figcaption>
+              </figure>
+            <section class="contacts">
+              <div class="contact">
+                <div class="contact__type">
+                  Phone(mts)
+                    <div class="contact__value">+375297635734</div>
+                </div>
+                <div class="contact__type">
+                  E-mail
+                    <div class="contact__value">rm.levsha@gmail.com</div>
```

Comment:
I recommend using link mailto. https://css-tricks.com/snippets/html/mailto-links/
Then this contact will be clickable and mail-manager app would be open on click, which is very handful for user.